packages/react-components/react-tabs/stories/Tabs/TabListDefault.stories.tsx

```diff
@@ -1,29 +1,263 @@
+import {
+  makeStyles,
+  mergeClasses,
+  shorthands,
+  tokens,
+  Button,
+  Menu,
+  MenuItem,
+  MenuList,
+  MenuPopover,
+  MenuTrigger,
+  Tab,
+  TabList,
+} from '@fluentui/react-components';
+import { useIsOverflowItemVisible, useOverflowMenu, Overflow, OverflowItem } from '@fluentui/react-components';
```

Comment:
You can put both of these under one import statement

```suggestion
import {
  makeStyles,
  mergeClasses,
  shorthands,
  tokens,
  useIsOverflowItemVisible,
  useOverflowMenu,
  Button,
  Menu,
  MenuItem,
  MenuList,
  MenuPopover,
  MenuTrigger,
  Overflow,
  OverflowItem,
  Tab,
  TabList,
} from '@fluentui/react-components';
```